packages/react-composites/src/composites/common/CaptionsBanner.tsx

```diff
@@ -62,13 +62,15 @@ export const CaptionsBanner = (): JSX.Element => {
           captionsBannerProps.isCaptionsOn && (
             <div className={containerClassName}>
               <Stack horizontalAlign="center">
-                <Stack.Item style={{ width: '50%' }}>
+                <Stack.Item style={{ width: props.isMobile ? '90%' : '50%' }}>
```

Comment:
Let's extra these 2 constants to the top of the component like 

const mobileViewBannerWidth = '90%';
const desktopVIewBannerWidth = '50%';